docs/how-ubuntu-is-made/concepts/version-strings.md

```diff
@@ -298,16 +308,12 @@ Due to that, in a Debian native package there is no `-debian_revision`.
 This continues into *native Ubuntu packages*, which also do not have a
 `-debian_revision`. But remember that the package namespace is shared between
 Debian and Ubuntu, therefore a native Ubuntu package `foo` of version `1.0`
-could be overwritten if Debian ever adds `foo` > `1.0` (if not blocked the newer
-according to `dpkg --compare-versions` will be synced).
-
-That would even happen automatically via the
-[auto-sync](https://canonical-ubuntu-packaging-guide.readthedocs-hosted.com/en/latest/explanation/debian-merges-and-syncs/#sync).
-To avoid this, it is recommended to add a `ubuntu0` suffix to the package.
+can be overwritten by Debian if they also add a package `foo` > `1.0`.
+To prevent this, add an `ubuntu` suffix to the package - otherwise {ref}`auto-sync <merges-syncs>` will use the newer version according to `dpkg --compare-versions`.
 
 We'd also like to be able to differentiate between "a native Debian package that
 got an Ubuntu Delta added" which could be `2.0ubuntu1`, and "a native Ubuntu
-package". Therefore the marker suffix for a native package shall be `ubuntu0`
+package". Therefore the marker suffix for a native package shall be `ubuntu`
```

Comment:
In this text block I'd expect something like "this used to be different, therefore you'll find some other versions still. Help us changing to the new format on updates to converge on the new form. Discussed here link-to-ml-discussion"